### ISSUE TITLE
Fix colors/alignment on OAuth screen

### ISSUE THREAD

**Issue by beaulebens** [woocommerce/woocommerce#29849]
**Prerequisites (mark completed items with an [x]):**
- [x] I have have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

**Describe the bug**
When granting permission via OAuth for another app to connect to a WooCommerce store, the screen has the old purple color, and some missing spacing/padding around the main logo.

**Expected behavior**
I expected the colors to be the newer, richer purple, and for there to be consistent spacing around the logo.

**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:
1. Attempt to connect your store to anything that allows for OAuth based connections (e.g. https://www.pirateship.com/)
2. See broken layout

**Screenshots**
![image-2](https://user-images.githubusercontent.com/108942/117368771-364d9380-ae81-11eb-9e5f-bc6612bd8a52.png)

**Comment by calvinmorett** [comment]
**Note**: Logo image is already updated

-------

### Solution:
![image](https://user-images.githubusercontent.com/11654917/126912152-37762dbe-21b7-47d1-aa6a-4795906fb224.png)

**auth.css**
- **Edit** margin from no margin top, to include a margin in the top, and the bottom. From `0 0 24px` to `24px 0 16px`

```
#wc-logo {
    border: 0;
    margin: 24px 0 16px;
    padding: 0;
    text-align: center;
}
```

**Comment by jorgeatorres** [comment]
Fixed in #39182.
